PageViewController.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  PageViewController.swift
+//  berkeley-mobile
+//
+//  Created by Eashan Mathur on 10/17/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+class PageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    
+    
+    var pages = [UIViewController]()
+    private let pageControl : UIPageControl = {
+        let pageControl = UIPageControl()
+        return pageControl
+    }()
+    var currentIndex: Int {
+        guard let vc = viewControllers?.first else { return 0 }
+        return pages.firstIndex(of: vc) ?? 0
+    }
+    var getStarted = UIButton()
+    let skipButton = UIButton()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.view.backgroundColor = .white
+        
+        self.dataSource = self
+        self.delegate = self
+
+        configurePageControl()
+
+        configureButtons()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        pageControl.subviews.forEach {
+            $0.transform = CGAffineTransform(scaleX: 1.5, y: 1.5)
+        }
+    }
+    
+    func configureButtons() {
+        print("Button should show")
+        let nextButton = UIButton()
+        nextButton.setTitle("Next", for: .normal)
+        nextButton.setTitleColor(.gray, for: .normal)
+        nextButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
+        view.addSubview(nextButton)
+        nextButton.addTarget(self, action: #selector(nextClicked(_:)), for: .touchUpInside)
+        nextButton.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
just a consistency thing but it might be better to create the button like you did with pageControl. same for the other buttons (and generally any other ui elements that are just initialized once). something like:
let nextButton: UIButton = {
        let button = UIButton()
        button.setTitle("Next", for: .normal)
        button.setTitleColor(.gray, for: .normal)
        button.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
        button.addTarget(self, action: #selector(nextClicked(_:)), for: .touchUpInside)
        button.translatesAutoresizingMaskIntoConstraints = false
return button
}()